test/JDBC/upgrade/14_7/schedule

```diff
@@ -366,4 +364,6 @@ app_name
 case_insensitive_collation
 sys-has_perms_by_name
 sys-has_perms_by_name-dep
+sys-all_sql_modules
+sys-sql_modules
```

Comment:
These changes look unnecessary to me, we are adding and removing the same tests.